toast/ios/Plugin/ToastLabel.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+
+@objc public class ToastLabel: UILabel {
```

Comment:
This should probably be internal, not public. There's no reason to expose it outside of the plugin.
```suggestion
@objc internal class ToastLabel: UILabel {
```